src/test/java/tictactoe/PlayerTypesTest.java

```diff
@@ -0,0 +1,23 @@
+package tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PlayerTypesTest {
+
+    @Test
+    void returnsHFromEnum() {
```

Comment:
Similar to my previous comment on the `GameOutcome` enum. However, I don't think these tests are needed at all. Why do you need to return `c` and `h` from the `COMPUTER` and `HUMAN` enums?